### DISCUSSION TITLE
versioning is not working correctly

### DISCUSSION THREAD

**Discussion by yana-khusainova** [thuliteio/doks#671] [Q&A]
Hi! 

I have a problem with versioning. I've made it like in example https://github.com/authelia/doks thanks to that person, who shared this :) but it doesn't work correctly

Versioning works only for the "prologue", the first section, here's the link on site https://authelia-staging.netlify.app/configuration/4.35/prologue/introduction/ (I'm really sorry if I shouldn't use another's example I just can't share mine). 

So the question is how can I do versioning for others sections. I write link for version like this `href="/{{ $current.Section }}/{{ $release.urlPrefix }}{{ $version.v }}/studio/studio/"` Can I change `/prologue/introduction/` on `/section/filename/` for examle to make it work for all sections?

Also I've noticed in this example https://github.com/h-enk/doks there's no folders of versions (like folder 0.1, folder 0.2) but it works correctly. But I don't understand this way at all :(

So could you help me with this changing "`/prologue/introduction/` on `/section/filename/`" I can't make up with how to name section and filename correctly.

Thank you in advance! You've made a really great theme!

**Comment by h-enk** [comment]
> Also I've noticed in this example https://github.com/h-enk/doks there's no folders of versions (like folder 0.1, folder 0.2) but it works correctly. But I don't understand this way at all :(

Branches are you used in the Doks default setup. James from Authelia is using a different setup: subfolders — see also https://github.com/h-enk/doks/discussions/621#discussioncomment-2010941 — but I think (as you can read in the thread) it's not stable yet. So, I'd go for the proven (Bootstrap Docs based) Doks default setup. 

A demo website is still in the making (I'm a bit busy) — a POC is available (maybe that can be of help):

- Repository: https://github.com/h-enk/doks-versioning-poc
- Website: https://doks-versioning-poc.netlify.app/

Also make sure to check out the documentation (but I guess you have): https://getdoks.org/docs/extensions/versioned-documentation/

**Reply by yana-khusainova** [reply]
got it. thank you for answering!
